doc/.gitignore

```diff
@@ -0,0 +1 @@
+backend_vars.asciidoc.newvars
```

Comment:
this file must not be ignored. See https://progress.opensuse.org/issues/34303

@asdil12 you might want to update the ticket

Reply:
OK, removed this one.